jarbas/api/views.py

```diff
@@ -30,6 +30,11 @@ def get(self, request):
         if self._bool_param('suspicions'):
             self.queryset = self.queryset.suspicions()
 
+        # filter receipt_url
+        receipt_url = self._bool_param('receipt_url')
```

Comment:
Hi @matheushf, sorry for my last message, i talked in another place and I feel that I confused you.
README tells that this param is called 'has_receipt', but here you call him of 'receipt_url'. So, pick one :smile: .
I think receipt_url is more like suspicions style, without verb. But has_receipt tells more. So.. i don't know hahaha